src/Containers/CircleTable/CircleTable.tsx

```diff
@@ -57,6 +57,10 @@ export interface ICircleTable {
      * @param selectedChildIndex - the array index for the table
      */
     onTableClick: (selectedChildIndex: number) => void;
+    /**
+     * Determines if the table is used in the toolbar or not
+     */
+    toolbarUse?: boolean;
```

Comment:
isHighlightedWhenFocused

Is the better prop name to use. Always is prefix for Boolean

Reply:
Sound good! I will fix it and make another PR